### PR TITLE
update deps, replace deprecated atomic init

### PR DESCRIPTION
**Pull request by olback**

### CODE REVIEW

File: nautilus-extension/src/menu_provider.rs [hunk 1]
```diff
@@ -211,7 +211,8 @@ fn connect_activate_signal(raw_menuitem: *mut NautilusMenuItem, activate_fn: uns
             Some(mem::transmute(activate_fn as *mut c_void)),
             data as *mut c_void,
             None,
-            GConnectFlags::empty()
+            G_CONNECT_AFTER
```
Comment (talklittle) [inline nautilus-extension/src/menu_provider.rs]: Thanks for the pull request!

What is `G_CONNECT_AFTER`? Old commented code line can be removed.

File: nautilus-extension/src/nautilus_module.rs [hunk 2]
```diff
@@ -139,7 +139,7 @@ impl NautilusModule {
         };
 
         unsafe {
-            let module_type = g_type_module_register_type(self.module, G_TYPE_OBJECT, name.as_ptr(), &info, GTypeFlags::empty());
+            let module_type = g_type_module_register_type(self.module, G_TYPE_OBJECT, name.as_ptr(), &info, G_CONNECT_AFTER);
```
Comment (talklittle) [inline nautilus-extension/src/nautilus_module.rs]: I don't think `G_CONNECT_AFTER` is a valid flag for `g_type_module_register_type`:

https://developer.gnome.org/gobject/unstable/GTypeModule.html#g-type-module-register-type

https://developer.gnome.org/gobject/unstable/gobject-Type-Information.html#GTypeFlags

`G_TYPE_FLAG_ABSTRACT` and `G_TYPE_FLAG_VALUE_ABSTRACT` are allowed.

Comment (olback) [reply nautilus-extension/src/nautilus_module.rs]: ~~Yes, this is correct. My bad. I'm not sure which value to use though. I'm guessing that `G_TYPE_FLAG_VALUE_ABSTRACT` would be correct here though.~~

Edit: `GTypeFlags::empty()` could be replaced with a here as well `0`.
https://docs.rs/bitflags/1.2.1/bitflags/#default

File: nautilus-extension/src/menu_provider.rs (hunk 1 above)
Comment (olback) [reply nautilus-extension/src/menu_provider.rs]: https://developer.gnome.org/gobject/stable/gobject-Signals.html#GConnectFlags

Seems like `GConnectFlags::empty()` actually just returns 0. So line 214 should really just be `0`.

**Comment by talklittle** [comment]
Thank you for the fixes.

Note to self and for posterity: the `GConnectFlags` and `GTypeFlags` were changed upstream in https://github.com/gtk-rs/sys/commit/df32145198d985d37d018883496b9c7bf975c96b from structs to primitive type aliases.

**Pull request merged**